distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/CatalogBundle.java

```diff
@@ -242,9 +256,14 @@ public void setupMaxDownloadRetryAttempts(int maxRetryAttempts) throws IOExcepti
 
     private void setConfigProperty(String pid, String propertyName, Object propertyValue)
             throws IOException {
-        Map<String, Object> existingProperties = Optional.ofNullable(adminConfig.getDdfConfigAdmin()
-                .getProperties(pid))
-                .orElse(new Hashtable<>());
+        Map<String, Object> existingProperties = null;
+        try {
+            existingProperties = Optional.ofNullable(adminConfig.getAdminConsoleService()
+                    .getProperties(pid))
+                    .orElse(new Hashtable<>());
+        } catch (NotCompliantMBeanException e) {
+            //ignore
+        }
         Hashtable<String, Object> updatedProperties = new Hashtable<>();
         updatedProperties.putAll(existingProperties);
```

Comment:
❗️ `existingProperties` may be null. We should skip this call to `putAll` in that case.